libs/utils/federated.js

```diff
@@ -1,59 +1,3 @@
-import { getConfig, SLD } from './utils.js';
```

Comment:
Do we have knowledge of anyone importing this file?

Reply:
Yes :( [here](https://github.com/adobecom/dme-partners/blob/17bb35bfd126882f9254857a3e00f09c82fe4365/edsdme/blocks/partners-navigation/utilities/utilities.js#L10)